content/blog/2024-05-01-almalinux-day-germany-recap.md

```diff
@@ -0,0 +1,110 @@
+
+### <u>The Strength of the AlmaLinux Community</u>
+
+The opening keynote for AlmaLinux Day: Germany shows us where AlmaLinux came from, and some of the amazing projects that have been using AlmaLinux in different ways. benny takes you through the history of AlmaLinux this far, including the exciting trends in adoption and who is engaged with the project, and then introduces the other speakers to share their stories.
+
+#### Speakers: 
+* benny Vasquez - Chair, AlmaLinux OS Foundation
+* Igor Seletskiy - CEO and Founder, Cloud Linux
+* Kenneth Tan, PhD - Executive Director, Sardina Systems
+* Hidenori Ishikawa - Manager / Product Development Engineer, Prime Strategy Co., Ltd.
+* Chieko Aihara - Vice President / Product Manager, Prime Strategy Co., Ltd.
+* Jonas Trüstedt - Senior Orcharhino Architect, ATIX AG
+* Tristan Theroux - IT Infrastructure & Security Director, SHED
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/P9pO0fPGVeo?si=P1rxAe4LWHywXQq2" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+language: English
+
+Chapters: 
+* [0:00](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=0s) Intro
+* [0:39](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=39s) The history of AlmaLinux
+* [14:44](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=884s) Igor Seletskiy from CloudLinux talks about where AlmaLinux came from and his investment in the future of AlmaLinux.
+* [18:39](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=1119s) Kenneth Tam from Sardina Systems talks about how they picked AlmaLinux as the base for their operations.
+* [27:55](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=1675s) Hidenori Ishikawa and Chieko Aihara from Prime Strategy talks about their incredible work in the CMS space with AlmaLinux as their base.
+* [34:06](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=2046s) Jonas Trüstedt from ATIX Consulting talks about why ATIX backs AlmaLinux and how Ocharino benefits from an open source ecosystem.
+* [41:41](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=2501s) Tristan Theroux from SHED talks about their move from CentOS Stream to AlmaLinux.
+* [47:11](https://www.youtube.com/watch?v=P9pO0fPGVeo&t=2831s) More community highlights
+
+### <u>Building without following: RHEL compatibility for AlmaLinux's Community</u>
+
+Finding our footing without using Red Hat Enterprise Linux has been an exciting venture, but our number 1 question is still "How do you build AlmaLinux?" In this presentation Andrew Lukoshko takes you through where AlmaLinux gets its code, how we confirm compatibility, and how we build.
```

Comment:
> but our number 1 question is still "How do you build AlmaLinux?" In

This sounds like it's *our* number one question.

> but the number one question we still receive is, "How do you build AlmaLinux?"

Or something of that ilk.